src/node.cc

```diff
@@ -3059,7 +3059,7 @@ static void DebugPortGetter(Local<Name> property,
 #if HAVE_INSPECTOR
   if (port == 0) {
     Environment* env = Environment::GetCurrent(info);
-    if (env->inspector_agent()->IsStarted())
+    if (env->inspector_agent()->io())
```

Comment:
yes, `IsStarted()` means its newed a `NodeInspectorClient client_`, its the `io_` that means its listening on a port.